GTG/core/tasks.py

```diff
@@ -563,7 +563,10 @@ def is_recurring(self) -> bool:
 
     @GObject.Property(type=bool, default=False)
     def has_date_due(self) -> bool:
-        return self._has_date_due
+        if not self._has_date_due and self.parent:
+            return self.parent._has_date_due
```

Comment:
```suggestion
            return self.parent.has_date_due
```
I think this works? It avoids accessing a protected member.
Same with the other one.

Reply:
Sounds good, changed